gpytorch/mlls/exact_marginal_log_likelihood.py

```diff
@@ -37,3 +37,9 @@ def forward(self, output, target, *params):
         # Scale by the amount of data we have
         num_data = target.size(-1)
         return res.div_(num_data)
+
+    def pyro_factor(self, output, target, *params):
+        import pyro
+        loss = self(output, target, *params)
+        pyro.factor("gp_mll", loss)
```

Comment:
isn't this the wrong sign?

Reply:
No, mll returns a positive log likelihood. We negate it manually in all examples (e.g. `loss = -mll(output, train_y)`